css/style.css

```diff
@@ -6,3 +6,131 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+
+ /*full page background color */
+ html {
+    background-color:rgb(221, 240, 240)
+ }
+
+ /*header styling*/
+ .header-container {
+    
+    position: fixed;
+    top: 0;
+    left: 0;
+    
+}
+header {
+
+    background-color:rgb(177, 245, 245);
+    top: 0;
+    display: flex;
+    width: 100%;
+    height: 100px;
+
+}
+/*logo style*/
+.logo {
+    display: flex;
+    margin-left: 0;
+    
+}
+.logo img {
+    max-width: 100%;
+    display: block;
+    height: 100%;
+    }
+
+    /* title*/
+h1 {
+    text-align: center;
+    justify-content:left ;
+}
+/* navbar style*/
+li {
+    list-style: none;
+}
+nav li {
+    display: inline;
+    padding: 2em;
+}
+nav {
+    display: flex;
+    justify-content: flex-end;
+    width: 100%;
+    text-align: center;
+    font-size: 20px;
+}
+
+nav a {
+    text-decoration: none;
+    color: brown;
+    
+}
+/*body style*/
+body {
+    font-size: 16px;
+    margin: 160px 60px 0 100px;
+    line-height: 1.5;
+}
+h3{
+    padding-left: 60px;
+}
+p {
+    text-align:left;
+    font-family: Cambria;
+    padding-left: 60px;
+}
+
+
+/* article style */
+article {
+    display: grid;
+    align-items: center;
+    justify-items: left;
+    
+
+}
+
+article:first-child {
+    background-color:rgba(253, 253, 253, 0.5);
```

Comment:
I think at the moment this is not working, the use of it is correct but you just need a space between `article` and the `:` so `article :first-child` :)

Reply:
Thank you Zein for your comments and showing me how to make it better !